_posts/2018-08-10-links-to-ddd-perth.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title:  "Links to DDD Perth talks"
+date:   2018-08-10
+tags: serverless cloud native pwa webapps
+author: William Sia
+image: img/native-mobile.png
+---
+
```

Comment:
I think maybe it needs an intro of some description...  
"On the 4th of August I presented at DDDPerth. The title of the talk was...."